Cargo.toml

```diff
@@ -41,8 +41,8 @@ rpath = false
 blake2 = { git = "https://github.com/near/near-blake2.git", version = "0.9.1", default-features = false }
 borsh = { version = "0.8.2", default-features = false }
 bn = { package = "aurora-bn", git = "https://github.com/aurora-is-near/aurora-bn.git", default-features = false }
-evm = { version = "0.27.0", default-features = false }
-evm-core = { version = "0.27.1", default-features = false }
+evm = { git = "https://github.com/aurora-is-near/sputnikvm", default-features = false }
+evm-core = { git = "https://github.com/aurora-is-near/sputnikvm", default-features = false }
```

Comment:
I don't think we should be using our own sputnikvm here. We should only stick with upstream...?

Reply:
While https://github.com/rust-blockchain/evm/pull/34 is not merged, we need to use ours, to keep track of the state in precompiles?

Reply:
Ah! I missed this. Ok yeah, makes perfect sense.